autoscaling-analyser.Rproj

```diff
@@ -0,0 +1,13 @@
+Version: 1.0
```

Comment:
is it ok to commit the .Rproj?

Reply:
That's a good question. I did some research and found this article about that: https://community.rstudio.com/t/should-rproj-files-be-added-to-gitignore/1269
The people on the article mainly agree with keeping the Rproj file on git.

Reply:
ok